src/www/actions/contact.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { push } from "connected-react-router";
+
+const handleResponse = (dispatch, data) => {
+  console.log(data);
+  if (data.error) {
+    console.log(data.error);
+    dispatch({
+      type: "ERROR",
+      error: data.error
+    });
+  } else {
+    dispatch({
+      type: "CONTACT",
+      user: data.user
+    });
+    dispatch(push("/contact"));
+  }
+};
+
+export const contact = (first_name, last_name,  email, question) => {
+  return dispatch => {
+    return axios
+      .post("/contact", {
```

Comment:
Turns out this is the wrong endpoint.

```suggestion
      .post("/contact/submit", {
```

Also, the parameters it accepts are `firstName, lastName, email, message`, so you'll have to rename the arguments to those in order for the query to be accepted by the backend. (instead of first_name/last_name and question)